compiler/qsc_linter/src/lints/ast.rs

```diff
@@ -0,0 +1,36 @@
+use super::{declare_lint, push_lint};
+use crate::linter::{ast::AstLintPass, Lint, LintLevel};
+use qsc_ast::ast::{BinOp, ExprKind, Lit};
+
+declare_lint!(
+    DoubleParens,
+    LintLevel::Warning,
+    "unnecesary double parentheses"
```

Comment:
Good point. Changed the message to `"unnecessary parentheses"`, but left the Lint name as `DoubleParens` since that's what it matches against in the AST, is that reasonable?